### DISCUSSION TITLE
Habilitar el control externo del componente Tabs

### DISCUSSION THREAD

**Discussion by dommirr** [TiendaNube/nimbus-design-system#226] [Q&A] [answered]
En New Admin, nos enfrentamos a una limitación inherente al componente Tabs. Este componente se presenta como un componente no controlado.

Esta característica, mientras que proporciona ciertas ventajas en términos de simplicidad y facilidad de uso, también conlleva una limitación significativa: una vez que el componente Tabs se renderiza por primera vez, no podemos activar otra pestaña desde el componente padre. La razón principal de esta limitación radica en que el estado de qué pestaña está activa o no se mantiene internamente en el componente Tabs y no está expuesto ni accesible desde fuera.

Esta limitación plantea una limitación particular para nuestro proyecto, donde nos encontramos con la necesidad de activar pestañas específicas desde el componente padre tal como se muestra en el siguiente video:

https://github.com/TiendaNube/nimbus-design-system/assets/43445281/72a43b7b-27b6-4811-847b-e3405cb3ec5b

_Este comportamiento se puede hacer a través de Nimbus 1._

¿Podemos modificar el componente Tabs para que permita un control externo? ¿O debemos explorar alternativas que satisfagan esta necesidad manteniendo el control interno del componente?

**Comment by joacotornello** [comment, marked as answer]
Solved in #282, `@nimbus-ds/components@5.10.0`
